editor/src/components/canvas/commands/adjust-css-length-command.ts

```diff
@@ -0,0 +1,244 @@
+import { number } from 'prop-types'
+import { isLeft, isRight, left } from '../../../core/shared/either'
+import * as EP from '../../../core/shared/element-path'
+import { emptyComments, jsxAttributeValue } from '../../../core/shared/element-template'
+import {
+  GetModifiableAttributeResult,
+  getModifiableJSXAttributeAtPath,
+  jsxSimpleAttributeToValue,
+  ValueAtPath,
+} from '../../../core/shared/jsx-attributes'
+import { ElementPath, PropertyPath } from '../../../core/shared/project-file-types'
+import * as PP from '../../../core/shared/property-path'
+import { EditorState, withUnderlyingTargetFromEditorState } from '../../editor/store/editor-state'
+import {
+  CSSNumber,
+  parseCSSPercent,
+  parseCSSPx,
+  printCSSNumber,
+} from '../../inspector/common/css-utils'
+import { applyValuesAtPath } from './adjust-number-command'
+import {
+  BaseCommand,
+  CommandFunction,
+  CommandFunctionResult,
+  PathMappings,
+  TransientOrNot,
+} from './commands'
+
+export interface AdjustCssLengthProperty extends BaseCommand {
+  type: 'ADJUST_CSS_LENGTH_PROPERTY'
+  target: ElementPath
+  property: PropertyPath
+  valuePx: number
+  parentDimensionPx: number | undefined
+  createIfNonExistant: boolean
+}
+
+export function adjustCssLengthProperty(
+  transient: TransientOrNot,
+  target: ElementPath,
+  property: PropertyPath,
+  valuePx: number,
+  parentDimensionPx: number | undefined,
+  createIfNonExistant: boolean,
+): AdjustCssLengthProperty {
+  return {
+    type: 'ADJUST_CSS_LENGTH_PROPERTY',
+    transient: transient,
+    target: target,
+    property: property,
+    valuePx: valuePx,
+    parentDimensionPx: parentDimensionPx,
+    createIfNonExistant: createIfNonExistant,
+  }
+}
+
+export const runAdjustCssLengthProperty: CommandFunction<AdjustCssLengthProperty> = (
+  editorState: EditorState,
+  pathMappings: PathMappings,
```

Comment:
You already removed `pathMappings` in https://github.com/concrete-utopia/utopia/pull/2136
Maybe `master` is not merged into this branch yet?

Reply:
fixed, thanks